docs/content/installation/upgrade_manual_openshift.md

```diff
@@ -34,4 +34,8 @@ When using the Red Hat® OpenShift® Container Platform, the CSI (Container Stor
 
 12. Check the **Overview** tab and that the **Controller Image Tab** and **Node Image Tag** are showing the most up-to-date version of the driver and the **Status** is _Running_.
 
+13. (Optional for initial host definer installation) If desired, create the host definer by clicking **Create Instance** to create the host definer (`IBMBlockCSI_hostdefiner`).
```

Comment:
@matancarmeli7 is `IBMBlockCSI_hostdefiner` also in the latest web console?

Reply:
I don't know what is in the create instance page (I don't think the there is `IBMBlockCSI_hostdefiner`)

Reply:
I think that I had put this in at one point as a placeholder and it got through approvals.
I'll update to `HostDefiner` as it is listed in the installation section (which was taken from screen captures)

Reply:
@rivkap could you revisit all the web console steps with someone from QA?
I suggest you do a live walkthrough session together. otherwise it's hard to see the process with still screenshots

Reply:
updated

Reply:
@rivkap this is about more than the `IBMBlockCSI_hostdefiner` placeholder (e.g. it seems there is no such word as "instance" there).
please see a live/recorded walkthrough, to make sure nothing else is missed

Reply:
and regardless, in general, please use more explicit placeholders (e.g. `<"something"-placeholder>`), so we could tell them apart from official content more easily

Reply:
Updated all installation / upgrade/ uninstall sections to do with web console
@oriyarde please resolve if possible. thanks

Reply:
LGTM. thanks!